src/components/CartContainer.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import CartItem from './CartItem';
+import { useGlobalContext } from '../context/Context';
+import Logo from '../img/logo.svg';
+import CloseIcon from '../img/close-icon.svg';
+
+function CartContainer() {
+    const { isOpen, cart, toggleOpenCloseCart, clearCart, total } =
+        useGlobalContext();
+
+    return (
+        <div className={isOpen ? 'cart-container show' : 'cart-container'}>
+            <div className="cart">
+                <header>
+                    <img src={Logo} alt="logo" />
+                    <img
+                        className="close-cart-btn"
+                        src={CloseIcon}
+                        alt="close icon"
+                        onClick={(_) => toggleOpenCloseCart('CLOSE_CART')}
```

Comment:
mislim da moze i ovako samo:
```suggestion
                        onClick={() => toggleOpenCloseCart('CLOSE_CART')}
```